docs/topics/native/native-faq.md

```diff
@@ -136,28 +136,10 @@ good practise to have the global state immutable. If for some reason you need a
 object, use the `@konan.ThreadLocal` annotation on the object. Also the `kotlin.native.concurrent.AtomicReference` class could be
 used to store different pointers to frozen objects in a frozen object and automatically update them.
 
-## How can I compile my project against the Kotlin/Native master?
+## How can I compile my project with unreleased versions of Kotlin/Native?
 
-One of the following should be done:
+First, please consider trying [preview versions](eap.md).
```

Comment:
I'm not sure how to properly use internal links. Please let me know if this is wrong.